### PR TITLE
enable swagger ui

### PR DESCRIPTION
**Pull request by deads2k**
@jwforres For Sam.  I don't know his nick.

After starting openshift, you can hit https://localhost:8443/swagger-ui/ for the swagger UI.

### CODE REVIEW

File: pkg/cmd/server/origin/master.go [hunk 1]
```diff
@@ -447,15 +452,22 @@ func (c *MasterConfig) Run(protected []APIInstaller, unprotected []APIInstaller)
 	open.Handle("/", handler)
 
 	// install swagger
+	// Expose files in third_party/swagger-ui/ on <host>/swagger-ui
+	fileServer := http.FileServer(&assetfs.AssetFS{Asset: kui.Asset, AssetDir: kui.AssetDir, Prefix: "third_party/swagger-ui"})
```
Comment (deads2k) [inline pkg/cmd/server/origin/master.go]: @smarterclayton this seems really hacky.  Right or wrong?

Comment (smarterclayton) [inline pkg/cmd/server/origin/master.go]: Yeah, you have to do a bunch of other stuff too.

Comment (smarterclayton) [inline pkg/cmd/server/origin/master.go]: Jessica gave me the rundown which I can give to you.

Comment (deads2k) [inline pkg/cmd/server/origin/master.go]: > Yeah, you have to do a bunch of other stuff too.

That'll teach me to stick my fingers where they don't belong. :)

Comment (smarterclayton) [inline pkg/cmd/server/origin/master.go]: Your enthusiasm is appreciated.  That'll be gone soon. :)

----- Original Message -----

**Comment by deads2k** [comment]
@spadgett

**Comment by deads2k** [comment]
I'm going to go back and hide behind the curtain now....

**Comment by smarterclayton** [comment]
When you stare into the abyss, sometimes it stares back.

----- Original Message -----

**Pull request closed without merging**